### PR TITLE
Adb/feature/style selector

### PR DESCRIPTION
**Pull request by adbarc92**
* Styles should display as circles
* Styles should be selectable

### CODE REVIEW

File: client/src/__tests__/utils.test.js [hunk 1]
```diff
@@ -27,26 +34,49 @@ describe('composeParams', () => {
...
-      "1": '-1',
-      "2": '1',
-      "3": '1',
-      "4": '3',
-      "5": '8'
+      1: '-1',
+      2: '1',
+      3: '1',
+      4: '3',
+      5: '8',
```
Comment (Jordan-Trimarchi) [inline client/src/__tests__/utils.test.js:53]: Should I be adding commas at the ends of objects?

File: client/src/components/overview/ProductOverview.jsx [hunk 2]
```diff
@@ -1,50 +1,32 @@
...
-          productPrice={product.default_price || ''}
-        />
-        <div>{JSON.stringify(productStyles)}</div>
-      </>
-      {/* ) : (
-        <div>Loading...</div>
-      )} */}
-    </div>
+    <>
+      <div>{JSON.stringify(reviewsMeta.ratings)}</div>
```
Comment (Jordan-Trimarchi) [inline client/src/components/overview/ProductOverview.jsx:17]: do we still need this?

**Review by Jordan-Trimarchi — APPROVED** [review]
styleSelector looks hecka clean. Can't say I totally get what it does. 
I see you've taken to trying to appease the linter

File: client/src/components/overview/ProductOverview.jsx (hunk 2 above)
Comment (adbarc92) [reply client/src/components/overview/ProductOverview.jsx:17]: No, I'll replace it with your stars when we get everything merged

**Pull request merged**